core/src/test/java/org/elasticsearch/action/bulk/TransportShardBulkActionTests.java

```diff
@@ -646,6 +655,199 @@ public void testPrepareIndexOpOnReplica() throws Exception {
         closeShards(shard);
     }
 
+    public void testProcessUpdateResponse() throws Exception {
```

Comment:
It's a test method which does not need Javadocs as it's not part of a public API. Sometimes we add explanatory comments to non-straightforward tests but this is not such a test.